modules/axtask/src/run_queue.rs

```diff
@@ -238,7 +255,8 @@ impl<'a, G: BaseGuard> AxRunQueueRef<'a, G> {
             can_preempt
         );
         if can_preempt {
-            self.inner.resched(true);
+            self.inner.put_prev(true);
```

Comment:
Could you help to add comments for `preempt_resched`?

Reply:
ofc

Reply:
see
 [Add comments for preempt_resched](https://github.com/arceos-org/arceos/pull/183/commits/e6997d195070205fcf7601ddd533ab8523a26f5a)